1-js/02-first-steps/12-while-for/1-loop-last-value/solution.md

```diff
@@ -8,18 +8,18 @@ while (i) {
 }
 ```
 
-Every loop iteration decreases `i` by `1`. The check `while(i)` stops the loop when `i = 0`.
+Cada iteração do loop subtrai `1` de `i`. O teste `while(i)` interrompe o loop quando `i = 0`.
```

Comment:
Don`t know if `subtrai` would be the best approach, considering the way you conjugate the _decrease_ word in the following sentence. I mean, you use `decresce`. Just a suggestion, but the translation its understandable. I think another person should check this.

Reply:
I understand. I suppose we could say 

> "Cada iteração do loop decresce `i` em `1`."

I find the phrasing with "subtrai" to sound more natural to Brazilian speakers, but I'll change it.